src/domain/hanke/hankeView/HankeView.tsx

```diff
@@ -339,27 +350,40 @@ const HankeView: React.FC<Props> = ({
 
       <InformationViewHeader backgroundColor="var(--color-summer-light)">
         <MainHeading>{hankeData?.nimi}</MainHeading>
-        <Text tag="h2" styleAs="h3" weight="bold" spacingBottom="l" data-testid="hanke-tunnus">
-          {hankeData?.hankeTunnus}
-        </Text>
-        <Text tag="p" styleAs="body-s" spacingBottom="l">
-          <strong style={{ marginRight: 'var(--spacing-s)' }}>
-            {t('hankePortfolio:labels:oikeudet')}:
-          </strong>
-          {t(`hankeUsers:accessRightLevels:${signedInUser?.kayttooikeustaso}`)}
-        </Text>
+        <Flex style={{ marginBottom: 'var(--spacing-m)' }} gap="4">
```

Comment:
Not a big thing but `marginBottom` could be defined straight as a prop without using style prop.